src/stories/design-system/IntroductionToCreatingStories.stories.mdx

```diff
@@ -0,0 +1,222 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
Kudos; this is good

Suggestion; wonder if we can update the component snippet file (plopfile.js) to generate this for us for new components?

Reply:
Here's the existing story template — https://github.com/GovConnex/UI/blob/ef7e4d44a28f2f554891b4d2403833174123f1f9/plop-templates/stories.tsx.hbs